lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -829,6 +833,23 @@ ConstExprFunctionState::computeCallResult(ApplyInst *apply) {
     setValue(apply, literal);
     return None;
   }
+  case WellKnownFunction::ArrayInitEmpty: { // Array.init()
+    assert(conventions.getNumDirectSILResults() == 1 &&
+           conventions.getNumIndirectSILResults() == 0 &&
+           "unexpected Array.init() signature");
+
+    auto literal = getConstantValue(apply->getOperand(1));
+    if (literal.getKind() != SymbolicValue::Metatype)
+      break;
+
+    auto literal_type = literal.getMetatypeValue();
+    SmallVector<SymbolicValue, 0> elementConstants;
```

Comment:
No need to create an array for this. `getArray` takes an `ArrayRef`, which can just be `{}`.